frontend/express/public/core/events/stylesheets/_main.scss

```diff
@@ -139,61 +196,76 @@
         font-weight: 300;
         line-height: 20px;
     }
-    &--green {
-        color: #12AF51;
-        padding-left: 8px;
-        font-size: 14px;
+    &__trend {
         font-weight: 500;
         line-height: 14px;
     }
-    &--red {
-        padding-left: 8px;
-        color: red;
-        font-size: 14px;
-        font-weight: 500;
-        line-height: 14px;
+    &__arrow {
+        padding-right: 3px;
     }
 }
 
-.cly-vue-events-all-name {
+.cly-vue-events-all {
+    &__sidebar {
+        .el-input__inner{
+            background-color: map.get(c.$colors, "warm-gray-20");
```

Comment:
you should not be overriding the core element components since the changes will be made in core and it will reflect the same everywhere.

Reply:
Sure. Will remove this